### DISCUSSION TITLE
How does 404.md work?

### DISCUSSION THREAD

**Discussion by Stwissel** [squidfunk/mkdocs-material#5153] [Q&A] [answered]
I added a 404.md to my site and ran into an issue (see below), so I tried to create a minimal reproducer, but there the 404.md was ignored and only the default rendered. So I wonder: what is needed (in mkdocs.yml I presume) to trigger a 404.md pickup?

The problem I ran into:
When the non-existing url contains a path, css and js aren't get picked up. So mysite.com/does-not-exist renders correctly. While mysite.com/this/does-not-exist fails. 

Identified reason: the js/css path in the rendered 404 are relative, not absolute. 

Help is very much appreciated

**Comment by squidfunk** [comment, marked as answer]
As far as I know, MkDocs doesn't support overriding the 404 page with an .md file – it must be a [static template](https://github.com/squidfunk/mkdocs-material/blob/0d124390c4f96188df0d3d27c867165bace0c965/src/mkdocs_theme.yml#L59-L60). This is not exclusive to Material for MkDocs, but to all MkDocs themes. Feel free to take this [upstream](https://github.com/mkdocs/mkdocs).

**Reply by Stwissel** [reply]
That's interesting. The repo I worked with didn't have a 404.md and I created one out of [curiosity|ignorance|serendipity] (pick one). And it worked. As suggested, I'll take it upstream.

**Reply by Stwissel** [reply]
Happy to report: 404.md is a working feature of mkdocs, I was able to create a [minimal reproducer](https://github.com/Stwissel/mknotfound) (rendered [here](https://github.com/Stwissel/mknotfound)).
Problem occurs in vanilla mkdocs already. [Reported upstream](https://github.com/mkdocs/mkdocs/issues/3140)

Thx for the help!
